ipatests/test_integration/test_installation.py

```diff
@@ -170,6 +170,37 @@ def test_replica2_with_ca_kra_install(self):
     def test_replica2_ipa_dns_install(self):
         super(TestInstallWithCA1, self).test_replica2_ipa_dns_install()
 
+    def test_install_with_bad_ldap_conf(self):
+        """
+        Test an client install with a non standard ldap.config
+        https://pagure.io/freeipa/issue/7418
+        """
+        ldap_conf = paths.OPENLDAP_LDAP_CONF
+        base_dn = self.master.domain.basedn  # pylint: disable=no-member
+        client = self.replicas[0]
+        tasks.uninstall_master(self.master)
```

Comment:
I don't understand why the test does uninstall + reinstall master?
Uninstall one of the replicas, then modify ldap.conf and try to install the node as client should be enough. Otherwise the test doesn't benefit from the already installed master and takes a long time to execute.

Reply:
Thank you for the feedback. Yes, the only replica uninstall is enough here.